### PR TITLE
docs: update android instruction

### PR DESCRIPTION
**Pull request by yshrsmz**
updated existing Android documentation.

### CODE REVIEW

File: docs/sdk/client-side/android/index.md [hunk 1]
```diff
@@ -416,3 +418,33 @@ This method will return null if the feature flag is missing in the SDK.
 
 </TabItem>
 </Tabs>
+
+
+### Listening to evaluation updates
```
Comment (yshrsmz) [inline docs/sdk/client-side/android/index.md:423]: added documentation for evaluation update listener

**Comment by cre8ivejp** [comment]
I just noticed that the v2 doesn't have the proguard file anymore.
If it is not necessary, we may delete this part.

https://github.com/bucketeer-io/bucketeer-docs/pull/6/files#diff-62188f6c279b29b97e1ef8a5a9f74017fe22bf51abc4c9771b4a78fcb962d2e4R37-R42

File: docs/sdk/client-side/android/index.md [hunk 2]
```diff
@@ -116,31 +117,35 @@ The initialize process starts polling the latest variations from Bucketeer in th
...
+val timeout = 1000 // Default is 5 seconds
+
+viewLifecycleOwner.lifecycleScope.launch {
+  val future = BKTClient.initialize(this.application, config, user, timeout)
+
+  // Future is blocking, so you need to wait on non-main thread.
+  val error = withContext(Dispatchers.IO) {
+    future.get()
+  }
+
```
Comment (cre8ivejp) [inline docs/sdk/client-side/android/index.md:136]: Please delete the space at the beginning of this line.

File: docs/sdk/client-side/android/index.md [hunk 3]
```diff
@@ -416,3 +418,33 @@ This method will return null if the feature flag is missing in the SDK.
 
 </TabItem>
 </Tabs>
+
+
+### Listening to evaluation updates
+
+BKTClient can notify when the evaluation is updated.
```
Comment (cre8ivejp) [inline docs/sdk/client-side/android/index.md:425]: Please delete the space at the end of this line.

Comment (yshrsmz) [reply docs/sdk/client-side/android/index.md:425]: these 2 spaces are needed to create a newline.

File: docs/sdk/client-side/android/index.md (hunk 2 above)
Comment (yshrsmz) [reply docs/sdk/client-side/android/index.md]: updated: [`f979f73` (#6)](https://github.com/bucketeer-io/bucketeer-docs/pull/6/commits/f979f73ee3689e16a8b1e03b8c76de20fed952fb)

File: docs/sdk/client-side/android/index.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ Implement the dependency in your Gradle file. Please refer to the [SDK releases
 
 ```groovy showLineNumbers
 dependencies {
-  implementation 'io.bucketeer:bucketeer-android-client-sdk:LATEST_VERSION'
+  implementation 'io.bucketeer:android-client-sdk:LATEST_VERSION'
```
[the remark this answers is not in the document]
Comment (yshrsmz) [reply docs/sdk/client-side/android/index.md:30]: updated: [`f979f73` (#6)](https://github.com/bucketeer-io/bucketeer-docs/pull/6/commits/f979f73ee3689e16a8b1e03b8c76de20fed952fb)

File: docs/sdk/client-side/android/index.md [hunk 5]
```diff
@@ -63,13 +63,13 @@ Configure the SDK config and user configuration.
...
-val config = BKTConfig.Builder()
+val config = BKTConfig.builder()
   .apiKey("YOUR_API_KEY")
-  .apiURL("YOUR_API_URL")
+  .endpoint("YOUR_API_URL")
   .featureTag("YOUR_FEATURE_TAG")
   .build()
 
-val user = BKTUser.Builder()
+val user = BKTUser.builder()
```
[the remark this answers is not in the document]
Comment (yshrsmz) [reply docs/sdk/client-side/android/index.md:72]: updated: [`f979f73` (#6)](https://github.com/bucketeer-io/bucketeer-docs/pull/6/commits/f979f73ee3689e16a8b1e03b8c76de20fed952fb)

**Pull request merged**